src/store/db.rs

```diff
@@ -271,13 +271,19 @@ fn serialize_outpoint(o: &OutPoint) -> Vec<u8> {
     v
 }
 
-fn vec_tx_seen_to_be_bytes(v: &[TxSeen]) -> Vec<u8> {
+fn vec_tx_seen_to_be_bytes(v: &[TxSeen]) -> Result<Vec<u8>> {
     let mut result = Vec::with_capacity(v.len() * VEC_TX_SEEN_SIZE);
     for TxSeen { txid, height, .. } in v {
         result.extend(txid.as_byte_array());
-        result.extend(height.to_be_bytes());
+        if *height >= 16777216 {
```

Comment:
is this ok for liquid?

rn we have 
```
 3 448 738 (2018 -> 2025) 
16 777 216
```
we could hit the max high if liquid decides to speed up blocks...